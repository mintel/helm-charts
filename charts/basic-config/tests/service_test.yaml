suite: Test Headless Service
templates:
  - service.yaml
release:
  namespace: test-namespace
tests:
  - it: Renders headless service
    set:
      global.name: test-app
      service:
        enabled: true
        headless: true
    asserts:
      - matchSnapshot: {} # Check for regressions and unexpected changes.
      - isKind:
          of: Service
      - equal:
          path: metadata.namespace
          value: test-namespace
      - equal:
          path: metadata.name
          value: test-app
      - equal:
          path: spec.clusterIP
          value: None
