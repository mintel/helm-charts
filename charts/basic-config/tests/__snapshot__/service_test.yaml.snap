Renders headless service:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        app.mintel.com/placeholder: placeholder
      labels:
        app.kubernetes.io/managed-by: Helm
        app.mintel.com/env: local
        app.mintel.com/region: local
        name: test-app
      name: test-app
      namespace: test-namespace
    spec:
      clusterIP: None
      selector: {}
      type: ClusterIP
